mindsdb/integrations/handlers/financial_modeling_prep_handler/README.MD

```diff
@@ -0,0 +1,80 @@
+# Financial Modeling Prep API Handler
```

Comment:
The queries in your examples use a table called `daily_chart_table`, but this does not seem to exist.